common.sh

```diff
@@ -20,8 +20,9 @@ TEST_GDR=${TEST_GDR:-0}
 get_opensuse1502_ami_id() {
     region=$1
     # OpenSUSE does not suppport ARM AMI's
+    # openSUSE-Leap-15.2 Build7.1 cabelo@opensuse.org
     aws ec2 describe-images --owners aws-marketplace \
-        --filters 'Name=name,Values=openSUSE-Leap-15.2-?????????-HVM-x86_64*' 'Name=ena-support,Values=true' \
+        --filters 'Name=product-code,Values=5080kaujzrzibjdwrkruspbj7' 'Name=ena-support,Values=true' \
```

Comment:
Is the product-code sufficient to identify openSUSE-Leap-15.2 on maketplace?

Reply:
Yes, I tried different regions and it returns the same openSUSE-Leap-15.2 AMI. We had done it for rhel 7* before.

Reply:
Okay~

Reply:
I have an additional check. I checked the opensuse ami used in current installer pipeline (which is not  broken because the last run is before yesterday) 
```
2020-12-02T18:43:38.059-08:00+ slave_value='ami-05a1be0cd8e9e9f8a ec2-user'
```
. It's the same ami id by querying via this product code. So it should not break installer pipeline.